pkg/controller/manager/manager_controller.go

```diff
@@ -296,7 +296,7 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 			svcDNSNames := dns.GetServiceDNSNames(render.ManagerServiceName, render.ManagerNamespace, r.clusterDomain)
 			svcDNSNames = append(svcDNSNames, "localhost")
 			certDur := 825 * 24 * time.Hour // 825days*24hours: Create cert with a max expiration that macOS 10.15 will accept
-			tlsSecret, err = utils.EnsureCertificateSecret(
+			tlsSecret, _, err = utils.EnsureCertificateSecret(
```

Comment:
I think now that we have this 2nd field we should make use of it for the manager secret also.

Reply:
Updated to use the EnsureCertificateSecret 2nd return value to check if certificate is operator managed